### PR TITLE
Add a env variable for genesis timestamp

### PR DESCRIPTION
**Pull request by AurelienFT**
Partially fix : #2423

### CODE REVIEW

**Review by AureliaDolo — DISMISSED** [review]
just the clippy warning and LGTM

(you can enable clippy warning in your favourite code editor)

**Review by AureliaDolo — APPROVED** [review]

**Review by adrien-zinger — APPROVED** [review]

File: massa-models/src/node_configuration/default.rs [hunk 1]
```diff
@@ -37,9 +37,11 @@ pub const CHANNEL_SIZE: usize = 256;
 lazy_static::lazy_static! {
     /// Time in millis when the blockclique started.
     pub static ref GENESIS_TIMESTAMP: MassaTime = if cfg!(feature = "sandbox") {
-        MassaTime::now()
-            .unwrap()
-            .saturating_add(MassaTime::from(1000 * 60 * 3))
+        std::env::var("GENESIS_TIMESTAMP").map(|timestamp| timestamp.parse::<u64>().unwrap().into()).unwrap_or_else(|_|
+            MassaTime::now()
+                .unwrap()
+                .saturating_add(MassaTime::from(1000 * 60 * 3))
```
Comment (adrien-zinger) [inline massa-models/src/node_configuration/default.rs:43]: Is it important?

Comment (adrien-zinger) [reply massa-models/src/node_configuration/default.rs:43]: nevermind :)

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**